nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -73,6 +74,8 @@ in
         name = dbUser;
         ensurePermissions = {
           "DATABASE ${dbName}" = "ALL PRIVILEGES";
+          "SCHEMA public" = "CREATE";
+          "ALL TABLES IN SCHEMA public" = "ALL PRIVILEGES";
```

Comment:
And this shouldn't be needed with the `preStart` script.

Reply:
Removed!